### PR TITLE
Adding CloudSqlMySql  and CloudSQLPostgreSQL connector

### PR DESCRIPTION
**Pull request by itsankit-google**
[PLUGIN-984](https://cdap.atlassian.net/browse/PLUGIN-984)

### CODE REVIEW

File: cloudsql-mysql-plugin/pom.xml [hunk 1]
```diff
@@ -43,9 +43,54 @@
...
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>hydrator-test</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-data-pipeline2_2.11</artifactId>
```
Comment (yaojiefeng) [inline cloudsql-mysql-plugin/pom.xml:61]: should this be in test scope?

File: cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorConfig.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+import java.util.Properties;
+import javax.annotation.Nullable;
+
+/**
+ * Configuration for CloudSQL MySQL connector
+ */
+public class CloudSQLMySQLConnectorConfig extends AbstractDBConnectorConfig {
+
+  private static final String JDBC_PROPERTY_CONNECT_TIMEOUT = "connectTimeout";
```
Comment (yaojiefeng) [inline cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorConfig.java]: just put units in the variable name so it is clear when reading the code, like `JDBC_PROPERTY_CONNECT_TIMEOUT_MILLIS`

File: cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLSink.java [hunk 3]
```diff
@@ -122,5 +105,10 @@ public String getTransactionIsolationLevel() {
       return ImmutableMap.of(
           CloudSQLMySQLConstants.CONNECTION_TIMEOUT, String.valueOf(connectionTimeout));
     }
+
+    @Override
+    protected CloudSQLMySQLConnectorConfig getConnection() {
```
Comment (yaojiefeng) [inline cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLSink.java:111]: Is this nullable? The connection won't have value at deploy time

File: cloudsql-mysql-plugin/src/test/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    connectionName = System.getProperty("connectionName");
+    Assume.assumeFalse(String.format(messageTemplate, "connectionName"), connectionName == null);
+  }
+
+  @Test
+  public void test() throws IOException, ClassNotFoundException, InstantiationException, IllegalAccessException {
+    test(new CloudSQLMySQLConnector(
+      new CloudSQLMySQLConnectorConfig(username, password, JDBC_PLUGIN_NAME, connectionArguments, instanceType,
+                                       connectionName, database)), JDBC_DRIVER_CLASS_NAME,
+         CloudSQLMySQLConstants.PLUGIN_NAME);
```
Comment (yaojiefeng) [inline cloudsql-mysql-plugin/src/test/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorTest.java]: nit- indentation seems off

File: cloudsql-postgresql-plugin/pom.xml [hunk 5]
```diff
@@ -48,11 +48,60 @@
...
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>hydrator-test</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-data-pipeline2_2.11</artifactId>
```
Comment (yaojiefeng) [inline cloudsql-postgresql-plugin/pom.xml:70]: should this be in test scope?

File: cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorConfig.java (hunk 2 above)
Comment (itsankit-google) [reply cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorConfig.java]: acknowledged.

File: cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLSink.java (hunk 3 above)
Comment (itsankit-google) [reply cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLSink.java:111]: ack, added annotation.

File: cloudsql-mysql-plugin/src/test/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorTest.java (hunk 4 above)
Comment (itsankit-google) [reply cloudsql-mysql-plugin/src/test/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorTest.java]: ack.

File: cloudsql-postgresql-plugin/pom.xml (hunk 5 above)
Comment (itsankit-google) [reply cloudsql-postgresql-plugin/pom.xml:70]: ack, scope added

File: cloudsql-mysql-plugin/pom.xml (hunk 1 above)
Comment (itsankit-google) [reply cloudsql-mysql-plugin/pom.xml:61]: ack, scope added.

**Review by yaojiefeng — APPROVED** [review]
LGTM, please make sure everything is tested before merging

**Pull request merged**